requirements.txt

```diff
@@ -1,6 +1,6 @@
 click
 daiquiri
-pipenv==2020.8.13
+pipenv
```

Comment:
should this be locked at 2018?

Reply:
Yes, I think so. You can do it in a separate PR.

Reply:
Sorry I added it here.

Reply:
Even better :+1: